source/fe/mapping.cc

```diff
@@ -49,6 +49,18 @@ Mapping<dim, spacedim>::get_vertices(
 
 
 
+template <int dim, int spacedim>
+void
+Mapping<dim, spacedim>::initialize_data(
+  const UpdateFlags                  update_flags,
+  const Quadrature<dim>             &quadrature,
+  std::unique_ptr<InternalDataBase> &data) const
+{
+  data = get_data(update_flags, quadrature);
```

Comment:
Doesn't the documentation say that it reuses the object rather than create a new one? I don't think this is wrong, it's just surprising to see it implemented this way :-)